src/FeatureBrowser/Cli/GenerateCommand.php

```diff
@@ -288,7 +288,7 @@ protected function makeOutputDir($directory)
         $path = $this->outputDirectory . $directory;
         if(!is_dir($path))
         {
-            mkdir($path, null, true);
+            mkdir($path, 0766, true);
```

Comment:
It also depends if people are running as the **owner** or **group** or **public**?

For me `0700` would work fine, but for others, they may need the **group** to have read/write permissions (not sure about public though). 

Although it should just be for `dev` mode, so not a major issue.